### PR TITLE
Add graph write options to playground

### PR DESCRIPTION
**Pull request by MingweiSamuel**

### CODE REVIEW

File: docs/src/pages/playground.js [hunk 1]
```diff
@@ -185,15 +185,29 @@ export function DatalogDemo() {
...
+    opShortText: false,
+  });
+  const writeGraphConfigOnChange = (name) => {
+    writeGraphConfig[name] = !writeGraphConfig[name];
+    setWriteGraphConfig({ ...writeGraphConfig });
+    return true;
+  };
 
-  const { output, diagnostics } = (compileFn)(program);
+  const { output, diagnostics } = (compileFn)(program, ...Object.values(writeGraphConfig));
```
Comment (shadaj) [inline docs/src/pages/playground.js:210]: Does JS guarantee that the order of keys is preserved? Not sure myself...

**Review by shadaj — APPROVED** [review]
nice!

File: docs/src/pages/playground.js (hunk 1 above)
Comment (MingweiSamuel) [reply docs/src/pages/playground.js:210]: good question

Comment (MingweiSamuel) [reply docs/src/pages/playground.js:210]: Yes but maybe worth making it less janky

Comment (MingweiSamuel) [reply docs/src/pages/playground.js:212]: Left it janky but added some comments

**Pull request merged**